apps/condo/domains/common/components/FeatureFlagsContext.tsx

```diff
@@ -0,0 +1,82 @@
+import { useAuth } from '@condo/next/auth'
```

Comment:
probably it's better to create `featureflags` folder and move everything to that folder

Reply:
Could you please provide the `README` with examples in that folder